engine/src/main/java/org/pentaho/di/trans/StepWithMappingMeta.java

```diff
@@ -55,7 +56,7 @@
  * @since 02-jan-2017
  * @author Yury Bakhmutski
  */
-public abstract class StepWithMappingMeta extends BaseStepMeta implements HasRepositoryDirectories {
+public abstract class StepWithMappingMeta extends BaseSerializingMeta implements HasRepositoryDirectories, StepMetaInterface {
```

Comment:
There are a handful of steps that already inherit StepWithMappingMeta.  Are we sure this won't break those steps?

Reply:
Pretty sure.  I looked at all of them, and none of them do a .super call for their implementations of those methods.

Reply:
:+1: